src/sqlfluff/utils/reflow/sequence.py

```diff
@@ -104,7 +104,9 @@ def get_raw(self) -> str:
 
     @staticmethod
     def _validate_reflow_sequence(elements: ReflowSequenceType):
-        assert elements, "ReflowSequence has empty elements."
+        # An empty set of elements _is_ allowed as an edge case.
+        if not elements:
+            return
```

Comment:
does this return want to be explicit?

Reply:
hmmm, perhaps yes I'll update that.

Reply:
Actually - I'm not sure. This function doesn't have a return value. @WittierDinosaur , do you know what best practice here is? I don't really want people to use a return value here, because it doesn't really have meaning (for instance, there's no return statement at the end of the method) - I just want to exit from the method early in the case that there's nothing else to check.

Reply:
you want to return `None`. That's the default `return` value of any function